diff --git a/home/flooki/Desktop/Embedded_Linux/Beagle-Bone/tmp/work-shared/beaglebone-yocto/kernel-source/arch/arm/boot/dts/am335x-bone-common.dtsi b/./am335x-bone-common.dtsi
index 0ccdc7cd463b..80c5261c8eb7 100644
--- a/arch/arm/boot/dts/am335x-bone-common.dtsi
+++ b/arch/arm/boot/dts/am335x-bone-common.dtsi
@@ -231,6 +231,12 @@ &i2c2 {
 	status = "okay";
 	clock-frequency = <100000>;
 
+	aht10@38 {
+	compatible = "aht10";
+	reg = <0x38> ; 
+	status = "okay";
+	};
+
 	cape_eeprom0: cape_eeprom0@54 {
 		compatible = "atmel,24c256";
 		reg = <0x54>;
